src/corredor/helpers/compose.ts

```diff
@@ -267,16 +350,16 @@ export class Compose {
    * @param {Record|Promise<Record>} record
    * @return {Promise<Record>}
    */
-  async saveRecord (record) {
+  async saveRecord (record: Record|Promise<Record>): Promise<Record> {
     return Promise.resolve(record).then(record => {
       if (!(record instanceof Record)) {
         throw Error('expecting Record type')
       }
 
       if (isFresh(record.recordID)) {
-        return this.ComposeAPI.recordCreate(record).then(r => new Record(record.module, r))
+        return this.ComposeAPI.recordCreate(record as unknown as KV).then(r => new Record(record.module, r))
       } else {
-        return this.ComposeAPI.recordUpdate(record).then(r => new Record(record.module, r))
+        return this.ComposeAPI.recordUpdate(record as unknown as KV).then(r => new Record(record.module, r))
```

Comment:
Would it make sense to make a shorthand fn? `kv(record)`

eg: `function kv(a: unknown): KV { a as KV }`